.github/workflows/uat.yml

```diff
@@ -26,8 +26,7 @@ jobs:
         with:
           java-version: 1.8
       - name: Build with Maven (not Windows)
-        working-directory: uat
-        run: mvn -ntp -U clean verify
+        run: mvn -DskipTests=true -ntp -U clean verify & mvn -U -ntp verify -f uat/pom.xml
```

Comment:
1. Seems first command rebuild  main project of Auth plugin - that does in other CI workflow, see maven.yml 

2. & run first comand in the shell background.

Reply:
fixed 736260a31d58a2c3a4a796482a041b170d3c1d37